consul/tombstone_gc_test.go

```diff
@@ -98,7 +98,7 @@ func TestTombstoneGC_Expire(t *testing.T) {
 
 	select {
 	case <-gc.ExpireCh():
-		t.Fatalf("shoudl be reset")
+		t.Fatalf("should be reset")
```

Comment:
Sure.  I considered saying "comments, docs, and strings," but some of the strings are parsed params that correlate to struct field names, I think.  Stuff like this is fine.